src/hotspot/cpu/aarch64/aarch64_vector_ad.m4

```diff
@@ -124,6 +124,9 @@ source %{
           (opcode == Op_VectorCastL2X && bt == T_FLOAT) ||
           (opcode == Op_CountLeadingZerosV && bt == T_LONG) ||
           (opcode == Op_CountTrailingZerosV && bt == T_LONG) ||
+          // Specially, the current vector implementation of Op_AddReductionVD/F works for
+          // Vector API only. If re-enabling them for superword, precision loss will happen
+          // because current generated code does not add elements sequentially from beginning to end.
```

Comment:
```suggestion
          // The vector implementation of Op_AddReductionVD/F is for the Vector API only. 
          // It is not suitable for auto-vectorization because it does not add the elements
          // in the same order as sequential code, and FPaddition is non-associative.
```